test/tests.js

```diff
@@ -1039,12 +1041,32 @@ var es2015 = function ES2015(ES, ops, expectedMissing, skips) {
 			var obj = { a: value };
 			defineProperty(obj, 'a', { configurable: false });
 
-			ES.Set(obj, 'a', value, true);
+			st.equal(ES.Set(obj, 'a', value, true), true, 'successful Set returns true');
 			st.deepEqual(obj, { a: value }, 'key is set');
 
 			st.end();
 		});
 
+		t.test('doesn’t call [[Get]] in conforming strict mode environments', { skip: noThrowOnStrictViolation }, function (st) {
```

Comment:
This needs to account for environments that don't support getters at all. I'll update it.